File: Try_GameEngine5/src/com/example/try_gameengine/scene/TransformSceneEffect.java
package com.example.try_gameengine.scene;

import com.example.try_gameengine.Camera.Camera;


public class TransformSceneEffect {
	Camera camera;
	public void shakeSceneEffect(Scene scene){
	 camera = scene.getCamera();
		
		
	}
	
	public void process(){
		float dx = 0;
		if(dx<10){
			camera.translate(dx, 0);
		}else{
			
		}
	}
}

File: Try_GameEngine5/src/com/example/try_gameengine/scene/TransformSceneEffect.kt
package com.example.try_gameengine.scene

import com.example.try_gameengine.Camera.Camera

class TransformSceneEffect {
    var camera: Camera? = null
    fun shakeSceneEffect(scene: Scene) {
        camera = scene.camera
    }

    fun process() {
        val dx = 0f
        if (dx < 10) {
            camera!!.translate(dx, 0f)
        } else {
        }
    }

    fun rotateSceneEffect(scene: Scene) {
        camera = scene.camera
    }

    fun process2() {
        camera!!.rotation(camera!!.rotation + 1)
    }
}
